radanalyticsio/spark-operator/base/clusterrolebinding.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRoleBinding
+metadata:
+  name: spark-operator-edit-resources
+roleRef:
+  kind: ClusterRole
+  name: cluster-admin
```

Comment:
Is this necessary? Can we create more specific role? We should not abuse the fact that Kubeflow has cluster-admin now

Reply:
Doh. You're right. I pulled this directly from radanalytics [operator.yaml](https://github.com/radanalyticsio/spark-operator/blob/master/manifest/operator.yaml) but it definitely doesn't need cluster-admin.